lesson15/main.js

```diff
@@ -0,0 +1,88 @@
+"use strict";
+const wrapper = document.getElementById("js-wrapper");
+const ul = document.getElementById("js-ul");
+const submitBtn = document.getElementById("js-submitBtn");
+submitBtn.classList.add("btn");
```

Comment:
nits
I think this should be controlled by CSS.

Reply:
Thank you for your comment.
I put a class on that button.
b0b0569